src/msw/webview_ie.cpp

```diff
@@ -51,6 +52,22 @@ enum //Internal find flags
 
 }
 
+// wxWebViewFactoryIE
+wxVersionInfo wxWebViewFactoryIE::GetVersionInfo()
+{
+    wxRegKey key(wxRegKey::HKLM, "Software\\Microsoft\\Internet Explorer");
+    wxString value;
+    key.QueryValue("Version", value);
+    long versions[3] = { 0, 0, 0 };
+    wxArrayString tokens = wxStringTokenize(value, ". ");
+    for (size_t i = 0; i < 3; i++)
+    {
+        if (tokens.size() > i)
+            tokens[i].ToLong(&versions[i]);
+    }
+    return wxVersionInfo("Internet Explorer", versions[0], versions[1], versions[2]);
```

Comment:
Let me try to write here my possibly more clear version just to check if you might not also prefer it:
```suggestion
    long major = 0,
         minor = 0,
         micro = 0;

    wxStringTokenizer tk(value, ". ");

    // Ignore the return value because if the version component is missing
    // or invalid (i.e. non-numeric), the only thing we can do is to ignore
    // it anyhow.
    tk.GetNextToken().ToLong(&major);
    tk.GetNextToken().ToLong(&minor);
    tk.GetNextToken().ToLong(&micro);

    return wxVersionInfo("Internet Explorer", major, minor, micro);
```